scripts/pipelines/azure-devops/templates/package/quarkus-package.sh

```diff
@@ -1,68 +1,33 @@
 #!/bin/bash
 ################################ Description ############################################
-# This script is used to build and push an image to a container registry
+# This script is used to build and push a quarkus project image to a container registry
 ################################# Arguments #############################################
-# -f : Path to the dockerfile
-# -c : The context of the build (in our case, the repository cloned and the build application are not in the same directory, we need to execute the docker build knowing that)
-# -u : Username to connect to the registry
-# -p : Password to connect to the registry
-# -r : Registry used to store the image
-# -i : Name of the image (containing the name of the registry and the path to the image)
-# -b : Name of the branch from where the version is coming
 # -t : Path to the pom.xml
 #########################################################################################
 set -e
 
-while getopts f:c:u:p:r:i:b:t:a:s:l: flag
+while getopts f:c:u:p:r:i:b:a:s:l:t: flag
 do
     case "${flag}" in
-        f) dockerFile=${OPTARG};;
-        c) context=${OPTARG};;
-        u) username=${OPTARG};;
-        p) password=${OPTARG};;
-        r) registry=${OPTARG};;
-        i) imageName=${OPTARG};;
-        b) branch=${OPTARG};;
+        f) ;; c) ;; u) ;; p) ;; r) ;; i) ;; b) ;; a) ;; s) ;; l) ;;
```

Comment:
```suggestion
        f) dockerFile=${OPTARG};;
        c) context=${OPTARG};;
        u) username=${OPTARG};;
        p) password=${OPTARG};;
        r) registry=${OPTARG};;
        i) imageName=${OPTARG};;
        b) branch=${OPTARG};;
        a) aws_access_key=${OPTARG};;
        s) aws_secret_access_key=${OPTARG};;
        l) region=${OPTARG};;
        t) pomPath=${OPTARG};;
```
see comment for the node package script

this solution finally does not work (sorry for that)
better define the variable in the parent script than in the common that is called inside